build-config/src/main/resources/weld-checkstyle/checkstyle.xml

```diff
@@ -50,9 +50,9 @@
             <property name="ignoreNumbers" value="-1, 0, 1, 2, 3"/>
         </module>
         <module name="MissingSwitchDefault" />
-        <module name="RedundantThrows">
+        <!--<module name="RedundantThrows">
```

Comment:
I had a bunch of checkstyle errors about `public` modifier in interfaces method for instance.